cmd/report.go

```diff
@@ -66,21 +57,23 @@ func createReport(execScheduler *execution.Scheduler, importedModules []string,
 		if !builtinOutputsIndex[o] {
 			continue
 		}
-		publicOutputs = append(publicOutputs, o)
+		_ = u.Strings("outputs", o) // TODO this will be moved and the error can be logged there
 	}
-
 	execState := execScheduler.GetState()
-	return report{
-		Version:    consts.Version,
-		Executors:  executors,
-		VUsMax:     execState.GetInitializedVUsCount(),
-		Iterations: execState.GetFullIterationCount(),
-		Duration:   execState.GetCurrentTestRunDuration().String(),
-		GoOS:       runtime.GOOS,
-		GoArch:     runtime.GOARCH,
-		Modules:    publicModules,
-		Outputs:    publicOutputs,
+	m, err := u.Map()
+
+	if m != nil {
+		m["k6_version"] = consts.Version
+		m["duration"] = execState.GetCurrentTestRunDuration().String()
+		m["goos"] = runtime.GOOS
+		m["goarch"] = runtime.GOARCH
+		m["goarch"] = runtime.GOARCH
+		m["goarch"] = runtime.GOARCH
```

Comment:
```suggestion
		m["goarch"] = runtime.GOARCH
```

I guess there's a couple extra ones 😛